### PR TITLE
[Merged by Bors] - chore(ModelTheory): Fix Model Theory docstrings

### PR DESCRIPTION
**Pull request by awainverse**
Updates the style on docstrings of files in the Model Theory folder to better match the current style guide.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/ModelTheory/Fraisse.lean [hunk 1]
```diff
@@ -19,31 +19,31 @@ Fraïssé limit - the countable ultrahomogeneous structure with that age.
...
+  fourth structure in `K` such that the resulting square of embeddings commutes.
 * `FirstOrder.Language.IsFraisse` indicates that a class is nonempty, isomorphism-invariant,
-essentially countable, and satisfies the hereditary, joint embedding, and amalgamation properties.
+  essentially countable, and satisfies the hereditary, joint embedding, and amalgamation properties.
 * `FirstOrder.Language.IsFraisseLimit` indicates that a structure is a Fraïssé limit for a given
-class.
+  class.
 
 ## Main Results
 * We show that the age of any structure is isomorphism-invariant and satisfies the hereditary and
```
Comment (YaelDillies) [inline Mathlib/ModelTheory/Fraisse.lean:36]: Mind also fixing this kind of stuff while you're at it?
```suggestion

* We show that the age of any structure is isomorphism-invariant and satisfies the hereditary and
```

Comment (awainverse) [reply Mathlib/ModelTheory/Fraisse.lean]: I've looked over the style guides again, and generalized to trying to match the current style.

File: Mathlib/ModelTheory/Algebra/Field/Basic.lean [hunk 2]
```diff
@@ -11,19 +11,18 @@ import Mathlib.Algebra.Field.MinimalAxioms
...
 
 ## Main definitions
-* `FirstOrder.Language.Theory.field` : the theory of fields
-* `FirstOrder.Model.fieldOfModelField` : a model of the theory of fields on a type `K` that
-  already has ring operations.
-* `FirstOrder.Model.compatibleRingOfModelField` : shows that the ring operations on `K` given
-by `fieldOfModelField` are compatible with the ring operations on `K` given by the
-`Language.ring.Structure` instance.
 
+- `FirstOrder.Language.Theory.field` : the theory of fields
```
Comment (YaelDillies) [inline Mathlib/ModelTheory/Algebra/Field/Basic.lean:20]: I thought people were normalising to using `*` for the outside bullet list? Maybe I'm wrong

Comment (awainverse) [reply Mathlib/ModelTheory/Algebra/Field/Basic.lean:20]: The official style guide uses `-`, for what I believe is still this case.
https://leanprover-community.github.io/contribute/style.html

I'm happy to use either, but I want a reference to follow.

**Comment by YaelDillies** [comment]
Thanks!

maintainer merge

**Comment by YaelDillies** [comment]
(since you're secretly not a new contributor at all)

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**